components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -274,7 +274,10 @@ private void updateProjectOrganizationName(String projectName) throws IOExceptio
         String templateFile = CmdUtils.getMicroGWConfResourceLocation() + File.separator
                 + CliConstants.BALLERINA_TOML_FILE;
         String fileContent = CmdUtils.readFileAsString(templateFile, false);
-        fileContent = fileContent.replace(CliConstants.MICROGW_HOME_PLACEHOLDER, CmdUtils.getCLIHome());
+
+        // Windows paths contains '\' separator which causes issues when included in ballerina.toml
+        String unixHomePath = CmdUtils.getCLIHome().replace('\\', '/');
```

Comment:
Why we need unixHomePath when using windows

Reply:
quoting from the issue,

> Generated value in the `Ballerina.toml` file looks like below. 
> ```
> "wso2/gateway" = { path = "C:\Users\ADMINI~1\toolkit\bin\../lib/gateway-balo/gateway-2019r3-java8-3.1.0.balo"}
> ```
> Replacing the backslashes in the windows path with forward slashes fixed the issue. This should be corrected in below line
> 
> https://github.com/wso2/product-microgateway/blob/4aa807bcc4bb9c01dd7f9c49d44044dfa4414890/components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java#L277

Ballerina get confused when windows style path is provided in ballerina.toml